test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareServiceTests.cs

```diff
@@ -1216,25 +1216,6 @@ public async Task VerifySchemaCompareServiceCalls()
                     DeploymentOptions = new DeploymentOptions()
                 };
 
-                await SchemaCompareService.Instance.HandleSchemaCompareRequest(schemaCompareParams, schemaCompareRequestContext.Object);
```

Comment:
Instead of just deleting (presumably) useful test code - could you instead add more logging so that if it does fail you can better identify where the failure is occurring?

Or is this not testing anything useful right now?

Reply:
There's already another test that tests schema compare cancellation that's been reliable, so testing it here doesn't add much value